src/schemas/components/SchemaFormPropertyMenu.vue

```diff
@@ -15,6 +17,7 @@
 
   const props = defineProps<{
     kind: PrefectKind,
+    disabled?: boolean,
```

Comment:
When a value is omitted you cannot change the prefect_kind. So adding the disabled prop hides all the kind options and just leaves the slot. Not super clean but works for now. Might revisit this if more things like this pop up.

Reply:
(non-blocking) - what does that look like? Just the label/description of the prop?

Reply:
scratch that I understand now - thought this was in the property component and not the menu